app/plugins/adp-backend/migrations/20240216160730_add_programme.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function up(knex) {
+  return knex.schema.createTable('delivery_programme', table => {
+    table.comment('Stores Delivery Programme data');
+    table
+      .string('programme_manager')
```

Comment:
Could potentially have more than 1 PDM. Should this be a one to many? May be an enhancement for later.